packages/fast-css-editor-react/src/editor.spec.tsx

```diff
@@ -79,4 +82,67 @@ describe("CSSEditor", () => {
 
         expect(callbackPosition).toHaveBeenCalled();
     });
+    test("should pass all position props to the CSSSpacing component", () => {
+        const topValue: string = "1";
+        const leftValue: string = "2";
+        const bottomValue: string = "3";
+        const rightValue: string = "4";
+        const callback: any = jest.fn();
+        const rendered: any = mount(
+            <CSSEditor
+                spacingType={SpacingType.padding}
+                marginTop={topValue}
+                marginLeft={leftValue}
+                marginBottom={bottomValue}
+                marginRight={rightValue}
+                paddingTop={topValue}
+                paddingLeft={leftValue}
+                paddingBottom={bottomValue}
+                paddingRight={rightValue}
+                onChange={callback}
+            />
+        );
+
+        const cssPosition: any = rendered.find("CSSSpacing");
+
+        expect(cssPosition.prop("spacingType")).toBe(SpacingType.padding);
+        expect(cssPosition.prop(SpacingKey.marginTop)).toBe(topValue);
+        expect(cssPosition.prop(SpacingKey.marginLeft)).toBe(leftValue);
+        expect(cssPosition.prop(SpacingKey.marginBottom)).toBe(bottomValue);
+        expect(cssPosition.prop(SpacingKey.marginRight)).toBe(rightValue);
+        expect(cssPosition.prop(SpacingKey.paddingTop)).toBe(topValue);
+        expect(cssPosition.prop(SpacingKey.paddingLeft)).toBe(leftValue);
+        expect(cssPosition.prop(SpacingKey.paddingBottom)).toBe(bottomValue);
+        expect(cssPosition.prop(SpacingKey.paddingRight)).toBe(rightValue);
+        expect(typeof cssPosition.prop("onSpacingUpdate")).toEqual(typeof Function);
+    });
+    test("should execute the `onChange` callback to recieve the correct CSSSpacing props", () => {
+        const updatedSpacingValue: string = "5";
+        const topValue: string = "1";
```

Comment:
why are the values of type string?  Could they be integers?

Reply:
they have to be strings because in the real world they would be 1px or 1em or 5% etc